### DISCUSSION TITLE
weird crash

### DISCUSSION THREAD

**Discussion by electrical** [Altinity/clickhouse-backup#166] [Q&A] [answered]
Hi,

Tried out the clickhouse-backup in 'server' mode and after a few minutes it crashed:

https://gist.github.com/electrical/4a72a20f914e2aaba6f0ff0ef34e4252

Any idea what could be causing this?
Running image version 0.6.4

**Comment by Slach** [comment, marked as answer]
sorry, miss discussions, 
i hope you can check latest 1.4.7 version which more stable
